src/lib/core/OperationalId.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <lib/core/NodeId.h>
+
+namespace chip {
+
+class OperationalId
```

Comment:
Is there  some documentation we could add  that is not obvious?

For myself when reading the class, I wonder "what is fabric index pointing into"?
There is a TODO below saying "replace fabricindex with fabricinfo pointer". This leads me to believe there is somewhere a fabricinfo table with fabricinfo data.

Describe that here: what does CHIP maintain (several fabrics), what is an index and how long is an index valid (does the operationalid ever become invalid? Can we remove fabrics? what if a fabric is removes and then a new one added? would a operationalid point to the newly created fabric because IDs are reused?)